lib/integrations/slack/link_unfurl_helper.rb

```diff
@@ -0,0 +1,53 @@
+module Integrations::Slack::LinkUnfurlHelper
+  def generate_unfurls(url, user_info)
```

Comment:
This method will generate blocks like [this](https://app.slack.com/block-kit-builder/T7NN675QT#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22fields%22:%5B%7B%22type%22:%22mrkdwn%22,%22text%22:%22*Name:*%5CnCandice%20Matherson%22%7D,%7B%22type%22:%22mrkdwn%22,%22text%22:%22*Email:*%5Cncandice@example.com%22%7D,%7B%22type%22:%22mrkdwn%22,%22text%22:%22*Phone:*%5Cn974237862323%22%7D,%7B%22type%22:%22mrkdwn%22,%22text%22:%22*Company:*%5CnChatwoot%22%7D%5D%7D,%7B%22type%22:%22actions%22,%22elements%22:%5B%7B%22type%22:%22button%22,%22text%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Open%20conversation%22,%22emoji%22:true%7D,%22value%22:%22https://d738-2406-7400-81-3754-8520-3004-f0a1-6c23.ngrok-free.app/app/accounts/1/conversations/100%22,%22action_id%22:%22button-action%22%7D%5D%7D%5D%7D).
<img width="544" alt="CleanShot 2023-09-19 at 12 24 03@2x" src="https://github.com/chatwoot/chatwoot/assets/12408980/e8ae22aa-8056-496f-af46-82ee66c6b6d4">